practice_app/pages/api/getTranslationModule.js

```diff
@@ -0,0 +1,803 @@
+exports.checkTranslationResponse = function (response, sourceLanguage, targetLanguage, originalText) {
+    const translation = response.data[0][0][0]
+    const sourceText = response.data[0][0][1]
+    const sourceLang = getSourceLangName(response.data[2])
+    const isAlternativeSourceExist = response.data[8] ? response.data[8][0].length > 1 : false
```

Comment:
What are these numbers for? for example what is response.data[8]?

Reply:
I think if alternative source languages for that texts exists, it is contained in response.data[8].

Reply:
This block is for gathering info from Google Translate response. I think the names of the constants makes it clear.